shared/components/SpeakerSocialLinks/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+
+const IconLink = ({ present, url, type, title }) => {
+  if (!present) { return null; }
+  return (
+    <a
```

Comment:
Currently ExternalLink requires you to provide children, there is no way of passing className with the background SVG, no title. This will require changing of the ExternalLink, which will require changing of all components that are using ExternalLink.